README.md

```diff
@@ -20,9 +20,10 @@ lein-ancient is destined for Leiningen >= 2.4.0.
 
 ## Usage
 
-```sh
-lein help ancient
-lein help ancient <subtask>
```

Comment:
Could you restore this? Lots of people don't seem to be aware of this feature in Leiningen, so it can't hurt to have a reminder here. :)

Reply:
Restored with 5fd556bbc755b36d1d882a3d74a3a729e9270100